python/src/aiconfig/eval/README.md

```diff
@@ -0,0 +1,112 @@
+# Tl;Dr
+
+Iterate on AIConfig faster. Measure AIConfig quality improvements and catch regressions. This module is closely related to [Promptfoo](https://www.promptfoo.dev/) and in fact it provides the option to integrate directly with Promptfoo. Please refer to their docs for motivating examples and usage details as needed.
+
+- You specify an AIConfig, test data, and evaluation metrics / assertions.
+- Each test case is a pair of (data, evaluation_metric).
+- The test data can either be AIConfig _inputs_ or precomputed _outputs_.
+- The evaluation metrics can be general, like string length, or have values that depend on the specific test case (like substring match, see example below).
+- Metrics are represented as floats, but can represent anything numerical: float/int, or any boolean condition (pass/fail).
+- AIConfig eval library runs them through your AIConfig (if necessary) and presents you a table to compare the results. Example:
+
+Inputs:
+
+```
+test_inputs_with_substrings = [
+    ("Food", "Magnolia Bakery"),
```

Comment:
Same food comment I made in https://github.com/lastmile-ai/aiconfig/pull/420/files#r1418150671

Let's just be sure to edit all the places where this was defined, including the notebook